src/test/auxiliary/blind-item-mixed-crate-use-item-foo.rs

```diff
@@ -8,8 +8,6 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-pub extern crate core; //~ ERROR: `pub` visibility is not allowed
+#![crate_type="lib"]
 
-fn main() {
-    pub use std::usize; //~ ERROR: imports in functions are never reachable
```

Comment:
Just confirming, but there is still a test for `pub use` in a function being an error, right?